app/src/androidTest/java/chat/rocket/android/settings/ui/SettingsFragmentTest.kt

```diff
@@ -1,113 +1,114 @@
-package chat.rocket.android.settings.ui
```

Comment:
What is your plan for this file?  Do you just want to delete it?  Or is this temporary?

Reply:
I had removed it temporarily as it is causing some tests to fails. I will update it with better tests soon.